quantum/pointing_device.h

```diff
@@ -80,7 +80,7 @@ void           pointing_device_task(void);
 void           pointing_device_send(void);
 report_mouse_t pointing_device_get_report(void);
 void           pointing_device_set_report(report_mouse_t newMouseReport);
```

Comment:
```suggestion
void           pointing_device_set_report(report_mouse_t report);
```
\+ on line 42 of the .c file.